ecs-cli/modules/cli/local/options/options.go

```diff
@@ -54,3 +56,21 @@ func ValidateFlagPairs(c *cli.Context) error {
 	}
 	return nil
 }
+
+// ContainerSearchInfo show the task definition filepath or arn/family message on local down.
+func ContainerSearchInfo(c *cli.Context) {
```

Comment:
Could we rename this to something like `ContainerDownSourceMessage` and have it just return a string (for ease of testing), and then when you call this in line 37 of `down_app` put the string returned here as a `logrus.Infof()` call?  That will also DRY up the code in this message.

Reply:
Done!